omod/src/main/webapp/resources/scripts/visit/visit.js

```diff
@@ -123,6 +123,7 @@ angular.module("visit", [ "filters", "constants", "encounterTypeConfig", "visitS
                             $http.get("/" + OPENMRS_CONTEXT_PATH + url)
                                 .then(function (response) {
                                     $scope.templateModel = response.data;
+                                    getFieldsWithValue($scope.templateModel.sections);
```

Comment:
I think we also need to handle forms with no sections, see: https://github.com/PIH/openmrs-module-pihcore/blob/master/omod/src/main/webapp/pages/visit/templates/encounters/defaultHtmlFormEncounterLong.gsp#L30